.github/workflows/ci.yaml

```diff
@@ -65,19 +65,19 @@ jobs:
       GOPATH: /home/runner/work/kpt-functions-sdk/go
       GO111MODULE: on
     steps:
-    - name: Set up Go 1.15
-      uses: actions/setup-go@v1
-      with:
-        go-version: 1.15
-      id: go
-    - name: Check out code into GOPATH
-      uses: actions/checkout@v1
-      with:
-        path: go/src/github.com/GoogleContainerTools/kpt-functions-sdk
-    - name: Build, Test, Lint
-      run: |
-        cd go
-        make all
+      - name: Set up Go 1.16
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.16
```

Comment:
should be `1.16.1` or `1.16.0-beta1`.
https://github.com/actions/setup-go#supported-version-syntax